Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -269,11 +269,9 @@ def add_learner_name_widget(self):
             orientation=Qt.Horizontal, callback=self.learner_name_changed)
 
     def add_bottom_buttons(self):
-        box = gui.hBox(self.controlArea, True)
-        box.layout().addWidget(self.report_button)
-        gui.separator(box, 15)
-        self.apply_button = gui.auto_commit(box, self, 'auto_apply', '&Apply',
-                                            box=False, commit=self.apply)
+        self.apply_button = gui.auto_commit(
```

Comment:
Neither this class nor class's children use attribute `apply_button`. Only the tests use it.